CatNote.API/Enums/TaskStatusDTO.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CatNote.API.Enums;
+
+public enum TaskStatusDTO
```

Comment:
It's not obvious what type this file stores. It's better for enum in such case to have name like '...Status', for example TaskDTOStatus